simulation_ws/src/aws_robomaker_simulation_common/nodes/route_manager

```diff
@@ -1,305 +0,0 @@
-        - output: bool
-            - false if noise, else true
-        '''
-
-        delta_x=max(2,self.meta_data.width//50) #to make it depend on resolution
-        delta_y=max(2,self.meta_data.height//50)
-
-        l_bound, r_bound = max(0, x-delta_x), min(self.meta_data.width-1, x+delta_x)
-        t_bound, b_bound = max(0, y-delta_y), min(self.meta_data.height-1, y+delta_y)
-
-        for _x in range(l_bound, r_bound):
-            for _y in range(t_bound, b_bound):
-                _row_id = self.ravel_index(_x, _y)
-                if self.occupancy_data.data[_row_id]!=0:
-                    return False
-
-        return True
-
-    def __iter__(self):
-        return self
-
-
-    def next(self):
-        '''
-        - description
-            for python 2.x support
-        '''
-        return self.__next__()
-
-
-    def __next__(self):
-        '''
-        - description:
-            Scans the map for a valid goal. 
-            Converts to world coordinates and wraps as a Pose to be consumed by route manager.
-        - input: 
-        - output:  
-                - Pose: 
-                      position: 
-                        x: double
-                        y: double
-                        z: double
-                      orientation: 
-                        x: double
-                        y: double
-                        z: double
-                        w: double
-        '''
-        z_world_floor = 0.
-        euler_orientation = [0.,0.,0.]
-
-        rospy.loginfo("Searching for a valid goal")
-        timeout_iter = 100
-        iteration = 0
-        while iteration<timeout_iter:
-          _x, _y = random.randint(0,self.meta_data.width-1), random.randint(0,self.meta_data.height-1)
-          _row_id = self.ravel_index(_x, _y)
-          if self.occupancy_data.data[_row_id]==0 and self.check_noise(_x, _y, row_id=_row_id): 
-            x_world, y_world  = self.grid_to_world_2d(_x, _y)
-            rospy.loginfo("Valid goal found!")
-            return self._create_pos(x_world, y_world, z_world_floor, *euler_orientation)
-
-        rospy.logerr("Could not find a valid goal in the world. Check that your occupancy map has 'Trinary' value representation and is not visually noisy/incorrect")
-        return None
-
-
-class RouteManager():
-    '''Send goals to move_base server for the specified route. Routes forever.
-
-       Loads the route from yaml. 
-       Use RViz to record 2D nav goals. 
-       Echo the input goal on topic /move_base_simple/goal
-
-       Format: 
-
-            order: inorder
-            poses: 
-                - pose: 
-                      position: 
-                        x: -5.41667556763
-                        y: -3.14395284653
-                        z: 0.0
-                      orientation: 
-                        x: 0.0
-                        y: 0.0
-                        z: 0.785181432231
-                        w: 0.619265789851
```

Comment:
Let's keep the more detailed comments.

Reply:
Not sure what that comment is trying to describe exactly.